starter-core/src/main/java/io/micronaut/starter/feature/lang/groovy/application.rocker.raw

```diff
@@ -34,11 +34,19 @@ class Application {
     static class DefaultEnvironmentConfigurer implements ApplicationContextConfigurer {
         @@Override
         public void configure(@@NonNull ApplicationContextBuilder builder) {
-            builder.defaultEnvironments("@(defaultEnvironment)");
+            builder.defaultEnvironments("@(defaultEnvironment)")
+@if (eagerInitSingleton) {
```

Comment:
I think it is best to remove this. We don't have any AOT feature which requires `eagerInitialization` to be specified via an `ApplicationContextConfigurer`.

Reply:
ping @melix